revocation/internal/crl/crl.go

```diff
@@ -119,7 +118,7 @@ func CertCheckStatus(ctx context.Context, cert, issuer *x509.Certificate, opts C
 		serverResults               = make([]*result.ServerResult, 0, len(cert.CRLDistributionPoints))
 		lastErr                     error
 		crlURL                      string
-		hasFreshestCRLInCertificate = hasFreshestCRL(cert.Extensions)
+		hasFreshestCRLInCertificate = x509util.FindExtensionByOID(oidFreshestCRL, cert.Extensions) != nil
```

Comment:
Why do we need `hasFreshestCRLInCertificate`? It's only used in a single `if` statement at line 138 below.

Reply:
Line 138 is inside the loop. We can compute it once and use it multiple times. Also, giving the value a name can make it clearer.